themes/default/content/docs/guides/crosswalk/aws/eks.md

```diff
@@ -147,6 +231,57 @@ Any existing tools will work here, including `kubectl`, Helm, and other CI/CD pr
 to define Kubernetes application-level objects and configuration in code too. For instance, we can deploy a canary
 to our EKS cluster in the same program if we want to test that it is working as part of `pulumi up`:
 
+{{< chooser language "javascript,typescript,python,go,csharp" / >}}
+
+{{% choosable language javascript %}}
```

Comment:
I'm not sure we have a hard and fast policy on this yet - but my own guidance has been that we should move toward not having both JavaScript and TypeScript examples in content like this.  The incremental value is fairly low, we are not consistent on it in many other places (most notably resource provider examples) so it sets users who do want it up for disappointment, and the maintenance burden of yet another manually managed copy is quite high.  I suggest we actually formalize this as our policy for mutli-language content.  Fine to leave these in here pending that decision if you'd prefer - but this was a reminder that we've avoided doing similar work in other places recently.

Reply:
For what it's worth, I actually assumed we'd go in the opposite direction, and maintain both, since TypeScript != JavaScript, and the latter has a larger community than the former. I agree for most examples the delta is tiny (although when it's material, it can be quite jarring to the uninitiated (generics and whatnot)) -- to me, it's honestly as much of a marketing thing, than being about the differences.